rdt/hyper_transformer.py

```diff
@@ -201,6 +201,7 @@ def update_transformers_by_sdtype(self, sdtype, transformer):
             transformer (rdt.transformers.BaseTransformer):
                 Transformer to be used for the given ``sdtype``.
         """
+        self.default_data_type_transformers[sdtype] = transformer
```

Comment:
@npatki If someone calls this method and then updates the `sdtype` of a column to match this `sdtype`, should the updated column use the old or new default transformer?